docs/apache-airflow-providers-apache-kafka/security.rst

```diff
@@ -0,0 +1,38 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Releasing security patches
+--------------------------
+
+Airflow providers are released independently from Airflow itself and the information about vulnerabilities
+is published separately. You can upgrade providers independently from Airflow itself, following the
+instructions found in :doc:`apache-airflow:installation/installing-from-pypi`.
+
+When we release Provider version, the development is always done from the ``main`` branch where we prepare
+the next version. The provider uses strict `SemVer <https://semver.org>`_ versioning policy. Depending on
+the scope of the change, Provider will get ''MAJOR'' version upgrade when there are
+breaking changes, ``MINOR`` version upgrade when there are new features or ``PATCHLEVEL`` version upgrade
+when there are only bug fixes (including security bugfixes) - and this is the only version that receives
+security fixes by default, so you should upgrade to latest version of the provider if you want to receive
+all released security fixes.
+
+The only exception to that rule is when we have a critical security fix and good reason to provide an
+out-of-band release for the provider, in which case stakeholders in the provider might decide to cherry-pick
+and prepare a branch for an older version of the provider following the
+`mixed governance model <https://github.com/apache/airflow/blob/main/PROVIDERS.rst#mixed-governance-model>`_
+and requires interested parties to cherry-pick and test the fixes.
```

Comment:
This was a deliberate choice, actually. I think we want a copy of security page for each provider. 

This text will be updated when we will publish SBOM information with links to SBOM information, so the content of this one **will** be different for every provider anyway (It has no variable part but it will be coming). 

It's very similar to those pages: 

https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/installing-providers-from-sources.html

Where links to .sha/downloaded tar.gz are generated automatically when documentation is published, but other than that the content will be the same.

I think it is important to have this information "in-provider" separately for each provider so that you do not have to go "elsewhere" to find the information and follow some "generic" advices - just to link directly to the information you need. 

> The issue is that every edit to this content will force handling doc release for all providers.

Correct. This is how it should be really, it's handled by "doc-only changes" option - this one will not release provider  itself but it will update the information in the latest released version of the provider. And once we add those links (I hope it will be before the next wave) we will have to updated them for all providers anyway (possibly even without actually releasing the providers because SBOM information is "post-processing" the releases - does not have to cause a new provider release. I am actually even planning to retroactively add such generated "security" pages back in all released version of the provider docs so that the user can see the SBOM information even in old versions of the docs.

Reply:
>This was a deliberate choice

this is enough explanation for me :)